Jenkinsfile

```diff
@@ -0,0 +1,151 @@
+@Library(value='pipeline-library@master', changelog=false)
+import com.genesys.jenkins.Service
+ 
+final serviceBuild = new Service()
+
+pipeline {
+
+    agent any
+
+    tools {
+        maven 'Maven 3.5.3'
+        jdk 'OpenJDK 1.8.0 (latest)'
+    }
+
+    environment {
+        NODE_VERSION = '11.15.0'
+        SNYK_TOKEN = credentials("PureCloudSnykToken")
+    }
+
+    stages {
+        stage("pre-build") {
+            steps {
+                copyArtifacts(projectName: 'build-github-swagger-codegen-sdkv1', filter: 'modules/swagger-codegen-cli/target/swagger-codegen-cli.jar', target: 'swagger-codegen', fingerprintArtifacts: true);
+
+                checkout scm
+
+                dir('platform-client-sdk-internal-config') {
+                    git url: "${scm.getUserRemoteConfigs()[1].getUrl()}"
+                }
+            }
+        }
+
+        stage("build") {
+            steps {
+                sh '''
+                    cd $WORKSPACE
+
+                    # Install NVM
+                    echo "About to run nvm.sh"
+                    source $HOME/.nvm/nvm.sh
+
+                    echo "nvm version: $(nvm --version)"
+
+                    echo "Installing node version $NODE_VERSION"
+
+                    nvm install $NODE_VERSION
+                    nvm use $NODE_VERSION
+
+                    npm install
+                    npm i -g snyk
+
+                    # Build SDKs
```

Comment:
wrap each sdk build command in a stage to identify the sdk. This will show build times per sdk on the job overview screen. Use names like "Java", "Java Guest", ".NET", "CLI", etc.